--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,5 @@
 AUTOMAKE_OPTIONS = subdir-objects
-
+ACLOCAL_AMFLAGS = -I m4
 AM_CFLAGS = -g -Wall
 
 PARSER_CFLAGS=@PARSER_CFLAGS@
@@ -41,7 +41,7 @@
 examples_active_LDADD = $(STROPHE_LIBS) 
 examples_roster_SOURCES = examples/roster.c
 examples_roster_CFLAGS = $(STROPHE_FLAGS)
-examples_roster_LDADD = $(STROPHE_LIBS) 
+examples_roster_LDADD = $(SSL_LIBS) $(STROPHE_LIBS)
 examples_basic_SOURCES = examples/basic.c
 examples_basic_CFLAGS = $(STROPHE_FLAGS)
 examples_basic_LDADD = $(STROPHE_LIBS) 
--- a/configure.ac
+++ b/configure.ac
@@ -2,6 +2,7 @@
 AM_INIT_AUTOMAKE([foreign])
 LT_INIT([dlopen])
 
+AC_CONFIG_MACRO_DIR([m4])
 AC_PROG_CC
 AM_PROG_CC_C_O
 PKG_PROG_PKG_CONFIG
